calour/io.py

```diff
@@ -432,131 +516,105 @@ def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_m
 
     Returns
     -------
-    Experiment
+    MS1Experiment
 
     See Also
     --------
     read
     '''
-    logger.debug('Reading OpenMS data (OpenMS bucket table %s, map file %s)' % (data_file, sample_metadata_file))
-    if rows_are_samples:
-        data_file_type = 'openms_transpose'
+    if file_type == 'mzmine2':
+        if rows_are_samples is None:
```

Comment:
you can set `False` as default

Reply:
again, i think maybe we'll need to set row_are_samples to True for openms, so need to be able to set based on data_file_type while still enable user override